app/controllers/bank_branches_controller.rb

```diff
@@ -4,7 +4,9 @@ def index
       @bank = Bank.find(params[:bank_id])
       @branches = []
     if params[:alphabetize] == 'true'
-      @branches = @bank.branches.order("name").all
+      @branches = @bank.branches.order('name').all
```

Comment:
I'm wondering if this is considered having query logic in the controller? I made mine a method in the model that I called right here, but I wasn't sure if that was overkill